ani-cli

```diff
@@ -299,7 +303,7 @@ play() {
 
 # setup
 agent="Mozilla/5.0 (Windows NT 6.1; Win64; rv:109.0) Gecko/20100101 Firefox/109.0"
-allanime_base="allanime.to"
+allanime_base="api.allanime.day"
```

Comment:
We're looking at two constants now, one for the "frontend" and one for the "api". You don't want to replace a bunch of strings, do you...

Reply:
it's now only api.allanime.day, cause allanime.day doesn't exist

Reply:
yes but it'd still be worth making into a variable because of how often it repeats

Reply:
allanime_base is still a good idea for a variable, yes.
Especially since we already have it, I dislike the current hardcoded find-replace

Reply:
I've implemented the above